basic_ds.go

```diff
@@ -166,6 +166,13 @@ func (d *LogDatastore) Delete(key Key) (err error) {
 	return d.child.Delete(key)
 }
 
+// DiskUsage implements the PersistentDatastore interface.
+func (d *LogDatastore) DiskUsage() (uint64, error) {
+	du, err := DiskUsage(d.child)
+	log.Printf("%s: DiskUsage: %d\n", d.Name, du)
```

Comment:
This should probably not return anything if an error was returned.

Reply:
The rest of methods there log the calls regardless of results. I can log the call and not the result though (just did that)